air/src/execution_step/execution_context/context.rs

```diff
@@ -75,7 +76,7 @@ pub(crate) struct ExecutionCtx<'i> {
     pub(crate) last_call_request_id: u32,
 
     /// Contains all executed results from a host side.
-    pub(crate) call_results: CallResults,
+    pub(crate) call_results: HashMap<u32, CallServiceResult>,
```

Comment:
It is unclear what is u32 key is for just looking at the struct so it is worth to elaborate on this in the comments.

Reply:
I would name `HashMap<u32, CallServiceResult>` type.

Reply:
Naming makes sense.

Reply:
Actually, I don't understand why it's a concern that one need to go to the definition of the type to see the key type? One can replace `CallResults` with explicit type, but it's used in many places in the code base, so there will be a lot of changes and the explicit type is much more verbose.

Reply:
@mikevoronov I don't quite understand what is your point.  `CallResult` is an interface type that now has `String` keys.